apps/webapp/src/components/header/header.tsx

```diff
@@ -27,18 +37,36 @@ export const Header = () => {
         </Link>
       </div>
       <Navbar />
-      <div className='flex w-full items-center justify-between md:w-auto md:gap-6 xl:gap-4'>
+      <div className='flex w-full items-center justify-between gap-4 md:w-auto md:gap-6 xl:gap-4'>
```

Comment:
We need a small gap for mobile, since we now have two icons right next to each other: the feedback icon and the bell icon